gammapy/scripts/find_obs.py

```diff
@@ -32,15 +60,150 @@ def find_obs(infile,
+             t_stop,
+             par_name,
+             par_min,
+             par_max,
+             invert,
              overwrite):
     """Select a subset of observations from a given observation list.
 
-    TODO: explain.
+    I still have a few TODO's to work out, but the script does filter
+    a self created dummy observation table and prints the output on
+    screen or saves it to a fits file.
+
+    Stringdoc still missing; for now, please have a look at
+    https://ms2.physik.hu-berlin.de/~mapaz/gammapy/docs/_build/html/obs/find_observations.html
+
+    For testing, use the following commands in your shell terminal:
+
+    gammapy-find-obs -h
+
+    gammapy-find-obs dummy.fits
+
+    gammapy-find-obs dummy.fits out.dummy.fits --overwrite
+
+    gammapy-find-obs dummy.fits --x 130 --y -40 --r 50 --system 'icrs'
+
+    gammapy-find-obs dummy.fits --x 0 --y 0 --r 50 --system 'galactic'
+
+    gammapy-find-obs dummy.fits --x 225 --y -25 --dx 75 --dy 25 --system 'icrs'
+
+    gammapy-find-obs dummy.fits --x -25 --y 0 --dx 75 --dy 25 --system 'galactic'
+
+    gammapy-find-obs dummy.fits --t_start '2012-01-01T00:00:00' --t_stop '2014-01-01T00:00:00'
+
+    gammapy-find-obs dummy.fits --par_name 'OBS_ID' --par_min 2 --par_max 6
+
+    gammapy-find-obs dummy.fits --par_name 'ALT' --par_min 60 --par_max 70
+
+    TODO: as for now, it doesn't really use the dummy.fits file;
+    the routine creates a random observation table with 100 entries
+    and filters on this. I need to produce a dummy.fits observation
+    table and put it in gammapy-extra. Once this is done, this should
+    be adapted consequently.
+
+    TODO: explain (edit this docstring) + link to gammapy/docs/_build/html/obs/find_observations doc.
+
+    TODO: update docs
+    old: https://gammapy.readthedocs.org/en/latest/obs/findruns.html
+    new: file:///home/mapaz/astropy/development_code/gammapy/docs/_build/html/obs/find_observations.html
+
+    TODO: write tests for this command-line tool!!!
     """
     import logging
     logging.basicConfig(level=logging.DEBUG, format='%(levelname)s - %(message)s')
-    # from gammapy.obs import RunList
 
-    # TODO: implement
-    raise NotImplementedError
\ No newline at end of file
+    if pix:
+        raise NotImplementedError
+
+    # open (fits) file and read the observation table
+    #observation_table = ObservationTable.read(infile)
+    # TODO: I need a test file in gammapy-extra!!!
+    #       for now: creating a test obs table REMOVE THIS!!!!
+    from ..datasets import make_test_observation_table
+    #observation_table = make_test_observation_table('HESS', 10)
+    observation_table = make_test_observation_table('HESS', 100)
+
+    # sky circle selection
+    do_sky_circle_selection = np.array([(x != None), (y != None),
+                                        (r != None), (system != None)])
+    if do_sky_circle_selection.all():
+        print("Applying sky circle selection") # TODO: use logging!!!
```

Comment:
@cdeil Since this is not an urgent problem (the regular output displayed is ok) (and I don't think you'll have time to spend on this tomorrow), I'm merging this now, and I pointed to this problem in the issue #315.